civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -10,33 +10,88 @@ use Drupal\civihr_employee_portal\Helpers\HelperClass;
  */
 function civihr_employee_portal_css_alter(&$css) {
 
-    // Remove defaults system messages css file.
-    unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+  // Remove defaults system messages css file.
+  unset($css[drupal_get_path('module', 'system') . '/system.messages.css']);
+
+  // Removing unwanted css files from civicrm pages.
+  if (civicrm_invoke()) {
```

Comment:
@AkA84 yes I'm using to check whether its an civicrm page. So you mean we can't use this function ?